contrib/devnet/download_snapshot.py

```
@@ -0,0 +1,118 @@
+        print("Removing existing cache...")
+        run_command(f'rm -rf "{SNAPSHOT_CACHE_DIR}"')
+
+    # Fetch snapshot info
+    print("\n[1/4] Fetching snapshot information...")
+    try:
+        snapshot_json = requests.get(SNAPSHOT_JSON_URL, timeout=30).json()
+        snapshot_link = snapshot_json['snapshots'][0]['link']
+        snapshot_filename = snapshot_json['snapshots'][0]['filename']
+        print(f"Snapshot: {snapshot_filename}")
+        print(f"Link: {snapshot_link}")
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)